src/main/resources/application-console.yml

```diff
@@ -38,7 +38,7 @@ spring:
         provider:
           keycloak:
             user-name-attribute: preferred_username
-            issuer-uri: http://${DOCKER_HOST_IP:172.17.0.1}:8082/auth/realms/contentcloud-gateway
+            issuer-uri: http://${DOCKER_HOST_IP:172.17.0.1}:8082/auth/realms/contentcloud-api
```

Comment:
Wasn't the realm going to be `contentcloud-management`?

Reply:
You asked not to change too many things at once. At the moment it's `-api` :grimacing:

Reply:
`-api` still seems sufficiently clear